com.minres.coredsl/src/com/minres/coredsl/validation/IssueCodes.xtend

```diff
@@ -1,7 +1,46 @@
 package com.minres.coredsl.validation
 
 class IssueCodes {
-	private new() {}
-	
-	public static final String SyntaxError = "org.eclipse.xtext.diagnostics.Diagnostic.Syntax";
+	private new() {
+	}
+
+	public static val SyntaxError = 'org.eclipse.xtext.diagnostics.Diagnostic.Syntax';
+
+	public static val _prefix = "com.minres.coredsl."
+
+	// utility issues
+	public static val DebugInfo = _prefix + 'DebugInfo';
+	public static val UnsupportedLanguageFeature = _prefix + 'UnsupportedLanguageFeature';
+
+	// type issues
+	public static val InvalidIntegerTypeSize = _prefix + 'InvalidIntegerTypeSize';
+
+	// isa issues
+	public static val DuplicateIsaStateElement = _prefix + 'DuplicateIsaStateElement';
+	public static val MismatchingIsaStateElementSignatures = _prefix + 'MismatchingIsaStateElementSignatures';
+	public static val MultipleIdenticalProvides = _prefix + 'MultipleIdenticalProvides';
```

Comment:
Really nice to have all these listed in a single place 👍

Reply:
It's also useful to quickly find the code responsible for emitting a certain error.